r/tests/testthat/test-dplyr-select.R

```diff
@@ -187,3 +186,32 @@ test_that("relocate with selection helpers", {
     df
   )
 })
+
+test_that("multiple select/rename and group_by", {
```

Comment:
I added these in because they are needed to make sure the implementation of the utility function `column_select()` is working properly.